cli/cmd/install/install.go

```diff
@@ -198,11 +221,52 @@ timescaledb-single:
 		helmValues = appendPrometheusHAValues(helmValues)
 	}
 
+	if c.enableOtel {
+		helmValues = enableOtelInValues(helmValues)
+	} else {
+		e, err := helmClient.ExportValuesFieldFromChart(c.Ref, c.ConfigFile, []string{"opentelemetryOperator", "enabled"})
+		if err != nil {
+			return err
+		}
+		var ok bool
+		c.enableOtel, ok = e.(bool)
+		if !ok {
+			return fmt.Errorf("enable otel was not a bool")
+		}
+	}
+
+	if c.enableOtel {
+		// opentelemetry operator needs cert-manager as a dependency as adding cert-manager isn't good practice and
+		// not recommended by the cert-manager maintainers. We are explicitly creating cert-manager with kubectl
+		// for more details on this refer: https://github.com/jetstack/cert-manager/issues/3616
+		apiClient := k8s.NewAPIClient()
+		_, err = apiClient.GetCRD("certificates.cert-manager.io")
+		if err != nil {
+			if errors.IsNotFound(err) {
+				fmt.Println("Couldn't find the cert-manager, Installing the cert-manager...")
+				utils.ConfirmAction()
+				err = k8s.CreateK8sManifests([]string{"https://github.com/jetstack/cert-manager/releases/download/v1.5.2/cert-manager.yaml"})
+				if err != nil {
+					return err
+				}
+				fmt.Println("Successfully created cert-manager")
+				time.Sleep(30 * time.Second)
+			} else {
+				return err
+			}
+		} else {
+			fmt.Println("cert-manager exists in the cluster, skipping the installation...")
+		}
+	}
```

Comment:
This block has a bit high cyclomatic complexity and we should think later on how to reduce it. Possibly abstracting cert-manager installation to a separate function and moving otel management to a pkg could help. But that can be an idea for later.